models/photos.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const photoSchema = new Schema(
+  {
+    name: { type: String },
```

Comment:
maybe photoName?

Reply:
For photoSchema, or for name? Do you think this will resolve the issue?

Reply:
resolved